docs/examples/transformations.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+title: Transformations
+parent: Examples
+nav_order: 4
+---
+# Transformations
+
+With transformations we can manipulate data prior to writing to the data store.
+
+## Aggregate
+With aggregates we can take multiple records and output a single record.
```

Comment:
In each of these, let's use the method name and format as code.  Ex: "`Aggregate` can take multiple records and output a single record."

Aggregate and Pivot work as written, but project and transform look like bad grammar.  By using method names we bypass the whole issue and I think it's more clear.